lib/stat_tracker.rb

```diff
@@ -19,4 +18,15 @@ def initialize(files)
     @game_team_collection.create_game_team_collection
   end
 
+  def highest_total_score
```

Comment:
Great way to save on lines of code! One-line syntax is something I should use too.